stat/distmat/wishart.go

```diff
@@ -145,19 +141,17 @@ func (w *Wishart) logProbSymChol(cholX *mat.Cholesky) float64 {
 	return 0.5*((fnu-fdim-1)*logdetx-tr-fnu*fdim*math.Ln2-fnu*w.logdetv) - mathext.MvLgamma(0.5*fnu, w.dim)
 }
 
-// RandSym generates a random symmetric matrix from the distribution.
-func (w *Wishart) RandSym(x *mat.SymDense) *mat.SymDense {
-	if x == nil {
-		x = &mat.SymDense{}
-	}
+// RandSymTo generates a random symmetric matrix from the distribution.
+// If dst is empty, it is resized to be an d×d symmetric matrix where d is the order
+// of the receiver. When dst is non-empty, RandSymTo panics if dst is not d×d.
+func (w *Wishart) RandSymTo(dst *mat.SymDense) {
 	var c mat.Cholesky
-	w.RandChol(&c)
-	c.ToSym(x)
-	return x
+	w.RandCholTo(&c)
+	c.ToSym(dst)
 }
 
 // RandChol generates the Cholesky decomposition of a random matrix from the distribution.
```

Comment:
RandCholTo

Also s/mat64/mat in the comment below.

Reply:
Done